src/fprime/fbuild/builder.py

```diff
@@ -311,6 +311,9 @@ def get_cmake_args(self) -> dict:
             if self.get_settings(setting, None) is not None
         }
 
+        # Load in the default settings
+        self.get_settings("default_cmake_options", None)
```

Comment:
@LeStarch this line doesn't seem to do anything, yet it can be misleading to think that it does (as this could potentially tweak the order in which arguments are loaded in). Can you confirm? If so, I'll remove it

Reply:
I was looking at this when reviewing https://github.com/fprime-community/fprime-tools/pull/165. Also I don't disagree with Josh's comment above but it may be a little late to switch